Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/common/DccJWKConverter.kt

```diff
@@ -37,4 +40,17 @@ class DccJWKConverter @Inject constructor() {
      * @return [PublicKey]
      */
     fun createPublicKey(jwk: DccJWK): PublicKey = createX509Certificate(jwk).publicKey
+
+    fun getCertificatePinner(
+        url: String,
+        jwkSet: Set<DccJWK>
+    ): CertificatePinner {
```

Comment:
This is where the list of allowed pairs is added, may work this way or not, never tested